### PR TITLE
feat(#217): add pathNotEq

### LINKED ISSUE #217: pathNotEq

### PR DESCRIPTION
**Pull request by BjornMelgaard**
Closes #217

### CODE REVIEW

**Comment by char0n** [comment]
Please rebase on top of master and drop yarn.lock file. Thanks

**Comment by BjornMelgaard** [comment]
@char0n done

**Comment by char0n** [comment]
You're still having some linting issues

File: src/index.d.ts [hunk 1]
```diff
@@ -590,6 +590,17 @@ declare namespace RamdaAdjunct {
             (value: any): (obj: object) => boolean;
         }
 
+        /**
+         * Determines whether a nested path on an object has not a specific value,
```
Comment (char0n) [inline src/index.d.ts]: has not -> doesn't have

File: src/pathNotEq.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import { pathEq, complement } from 'ramda';
+
+
+/**
+ * Determines whether a nested path on an object has not a specific value,
```
Comment (char0n) [inline src/pathNotEq.js]: has not -> doesn't have

File: src/pathNotEq.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+ * in R.equals terms. Most likely used to filter a list.
+ *
+ * @func pathNotEq
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.3.0|v2.3.0}
+ * @category Relation
+ * @sig  [k] → a → {a} → Boolean
```
Comment (char0n) [inline src/pathNotEq.js]:
```js
@sig [Idx] => a => {a} => Boolean
@sig Idx = String | Int

File: src/pathNotEq.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+ * @param {Array} path The property paths to fetch
```
Comment (char0n) [inline src/pathNotEq.js]: The path of the nested property to use

File: src/pathNotEq.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+ * @param {a} val The value to compare to
```
Comment (char0n) [inline src/pathNotEq.js]: The value to compare the nested property with

File: src/pathNotEq.js [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+ * @param {Object} object The object, that presumably contains value under the path
```
Comment (char0n) [inline src/pathNotEq.js]: The object to check the nested property in

File: src/pathNotEq.js [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+ * @return {Boolean} Comparison result
```
Comment (char0n) [inline src/pathNotEq.js]: Returns Boolean `false` if the value equals the nested object property, `true` otherwise

File: src/pathNotEq.js [hunk 8]
```diff
@@ -0,0 +1,29 @@
+import { pathEq, complement } from 'ramda';
+
+
+/**
+ * Determines whether a nested path on an object has not a specific value,
+ * in R.equals terms. Most likely used to filter a list.
+ *
+ * @func pathNotEq
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.3.0|v2.3.0}
```
Comment (char0n) [inline src/pathNotEq.js]: 2.4.0

File: test/pathNotEq.js [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+  });
+
+  it('tests example', function() {
+    const user1 = { address: { zipCode: 90210 } };
+    const user2 = { address: { zipCode: 55555 } };
+    const user3 = { name: 'Bob' };
+    const users = [user1, user2, user3];
+    const isFamous = RA.pathNotEq(['address', 'zipCode'], 90210);
+    const result = R.filter(isFamous, users); //= > [ user2, user3 ]
+    eq(result, [user2, user3]);
```
Comment (char0n) [inline test/pathNotEq.js]: insert one empty line pls

File: src/pathNotEq.js (hunk 3 above)
Comment (BjornMelgaard) [reply src/pathNotEq.js]: I've thought we can't do that

Comment (char0n) [reply src/pathNotEq.js]: We already can. I have added support some time ago https://github.com/char0n/docdash/commit/bf4b4eebfdaf042c0ea9038419bc26813f243c94

**Comment by BjornMelgaard** [comment]
@char0n done

**Comment by char0n** [comment]
Nice work!

**Pull request merged**